.github/workflows/release-step-1_manual_create-release-pr.yml

```diff
@@ -34,8 +36,15 @@ jobs:
           token: ${{ secrets.PA_TOKEN }}
           commit-message: One App Prepare Release
           author: 'OneAmexBot <one.amex@aexp.com>'
-          title: 'One App Prepare Release'
+          title: 'One App Prepare Release (${{ github.ref_name }})'
           body: 'This is an auto-generated pull request for One App release changes. Please review this PR and merge it for a release to be initiated. Please close this PR to stop a release from being initiated.'
           labels: release,automated pr,skip-release-drafter
-          branch: prepare-release
-          base: main
+          branch: ${{ format('prepare-release-for-{0}', github.ref_name)}}
+  validate-branch:
```

Comment:
what about having this go before `release-step-1_manual_create-release-pr` then using `needs:  validate-branch` to ensure that its always checked.

Reply:
Both block have opposite if statements. Meaning that the `release-step-1_manual_create-release-pr` checks if it should run without the need to rely on `validate-branch`. That said i have changed the order because that makes reading the code more sane